docs/code/best_practices.md

```diff
@@ -0,0 +1,28 @@
+# Best Practices
+
+MDC Web follows naming and documentation best practices to keep our code
+consistent, and our APIs user friendly. We follow isolation best practices to
+keep our code loosely coupled. And we follow performace best practices to keep
+our components fast.
+
+### Naming
+
+*  Always match [spec](https://material.io/guidelines)
```

Comment:
maybe: "Match spec whenever possible. If the nomenclature used in spec conflicts with a natively implemented element or pattern, reach out for guidance." <~maybe link to discord here